csp-guru/datapackage.json

```diff
@@ -105,56 +125,56 @@
                     },
                     {
-                        "name": "LCOE_10_2015EUR",
-                        "description": "levelized cost of electricity with 10% weighted average cost of capital [ct/kWh], 2015 EURO discounted"
+                        "name": "LCOE_10_25_[USD2017]",
+                        "description": "levelized cost of electricity with 10% weighted average cost of capital [USD2017ct/kWh] and a 25 year payback period, cap dependent O&M, deflated from Year_operational; if station under development or constrution then deflated from 2017"
                     },
                     {
-                        "name": "Other_support",
-                        "description": "financial assistance to project developer: tax credits, cheap credits, etc."
+                        "name": "LCOE_5_25_[EURO2017]",
+                        "description": "levelized cost of electricity with 5% weighted average cost of capital [EURO2017ct/kWh] and a 25 year payback period, deflated from Year_operational; if station under development or constrution then deflated from 2017"
                     },
                     {
-                        "name": "Owner",
-                        "description": "owner of the CSP plant"
+                        "name": "LCOE_10_25_[EURO2017]",
+                        "description": "levelized cost of electricity with 10% weighted average cost of capital [EURO2017ct/kWh] and a 25 year payback period, deflated from Year_operational; if station under development or constrution then deflated from 2017"
                     },
                     {
-                        "name": "Support_duration",
-                        "description": "duration of PPA, FiT, etc. [yr]"
+                        "name": "Concessional_funding;other_support",
+                        "description": "financial assistance to project developer: tax credits, cheap credits, etc."
                     },
                     {
-                        "name": "Enipedia_ID",
-                        "description": "https://enipedia.org/wiki/Enipedia"
+                        "name": "Owner",
+                        "description": "owner of the CSP plant"
                     },
                     {
-                        "name": "SolarPACES_ID",
-                        "description": "SolarPACES_ID/NREL: https://www.nrel.gov/csp/solarpaces/index.cfm "
+                        "name": "PPA_or_support_duration_[yr]",
+                        "description": "duration of PPA, FiT, etc. [yr]"
                     },
                     {
                         "name": "Sources",
-                        "description": "indicates where the information for each plant was taken from"
+                        "description": "indicates where the information on each plant was retrieved from, Lillestam et al 2017: Nature energy indicates the 2016-08-30 version of the CSP.guru dataset"
```

Comment:
CRITICAL!!!!11!!111!
Here, and _everywhere_ in the csv there is a typo in Lilliestam.

Reply:
luckily, it wasn't out typo.